### ISSUE TITLE
wallet  freeze , by selecting the amount of payment in the QR code

### ISSUE THREAD

**Issue by Exteroqs** [vergecurrency/VERGE#713]
Hi.
I have a problem when using the option to select quantity to request payment, the wallet is frozen. The problem often happens to me, i don't know if it only happens to me. I follow these steps (as in the image) and I do them with 1 sec between each step and at the moment of entering the amount the wallet freezes.
![screenshot_20180414_032037](https://user-images.githubusercontent.com/37254547/38765303-39c68dd6-3f95-11e8-878a-f63a35125041.png)

**Comment by marpme** [comment]
Will test it, right now on Ubuntu 16.04 LTS ... for edit later.

**Comment by Exteroqs** [comment]
I have compiled the latest version (until, Commits on April 6, 2018) of the master branch.
Maybe it happens on my desk kde. Is there a way to execute the wallet in debug mode?

**Comment by marpme** [comment]
@Exteroqs `./src/qt/VERGE-qt -debug -printtoconsole`

**Comment by Exteroqs** [comment]
Thanks, i will try know what happens

**Comment by Exteroqs** [comment]
mmm, I had not tried that option in previous versions.

**Comment by jrsikors** [comment]
I was just probing to see if it worked in 4.0.1 but broke in 4.0.2.

If it's possible it never worked right -- yeah I don't know what to say ! :(

**Comment by Exteroqs** [comment]
I tried in debug mode but it throws me only information of the block chains,  if I noticed that when the wallet was frozen, the program was still working connected to the network, only the GUI was frozen

**Comment by Exteroqs** [comment]
I was looking for the files related to the process of creating the QR code, the "bitcoinunits.cpp" file, the switch statements do not have the "brake;" at the end. that can generate problems on some occasions, I will try to compile it with "break;". in the file "bitcoinamountfield.cpp" I thought to add other conditions to the "bool BitcoinAmountField :: validate ()" function that could give problems in null values.
i compile and see what happen, if not happend nothing, well, i dont know, I suspect that it goes into a kind of loop somewhere

**Comment by Exteroqs** [comment]
No, it does not work :(, at least I tried it haha, I leave it to the experts
